lldb/source/Plugins/SymbolFile/DWARF/DWARFUnit.cpp

```diff
@@ -663,6 +663,30 @@ DWARFUnit::GetDIE(dw_offset_t die_offset) {
   return DWARFDIE(); // Not found
 }
 
+llvm::StringRef DWARFUnit::PeekDIEName(dw_offset_t die_offset) {
+  DWARFDebugInfoEntry die;
+  if (!die.Extract(GetData(), this, &die_offset))
+    return llvm::StringRef();
+
+  // Does die contain an AT_Name?
```

Comment:
```suggestion
  // Does die contain a DW_AT_name?
```
makes grepping more reliable